README.md

```diff
@@ -0,0 +1,8 @@
+# Remote-Settings-Client
+Remote-Settings Client (read-only) built using Rust to fetch records, collections and comes with signature verification
```

Comment:
I think this would be enough: `Remote Settings client built in Rust to fetch collection content.`

The rest (read-only, signature verification) could be listed in features below